pages/app/[slug].tsx

```diff
@@ -19,9 +20,22 @@ const AppSlug = () => {
     setSidebarOpen(!sidebarOpen);
   };
 
+  useEffect(() => {
+    void (async () => {
+      setIsLoading(true);
+      const data = await Service.require(
+        "subscriptions"
+      ).getActiveSubscription();
+      setIsLoading(false);
+      if (!data?.id) {
+        router.push("/app/subscriptions");
+      }
+    })();
+  }, [router.asPath]);
```

Comment:
We moved this logic to the component `auth-client-provider`

Reply:
I checked for the file `auth-client-provider` and I could not find this file, it seems its not on the main branch.